runelite-client/src/main/java/net/runelite/client/plugins/antidrag/AntiDragPlugin.java

```diff
@@ -113,12 +111,12 @@ public void keyTyped(KeyEvent e)
 	@Override
 	public void keyPressed(KeyEvent e)
 	{
-		if (e.getKeyCode() == KeyEvent.VK_CONTROL && config.disableOnCtrl() && !(inPvp || config.onShiftOnly()))
+		if (e.getKeyCode() == KeyEvent.VK_CONTROL && config.disableOnCtrl())
```

Comment:
I think this is wrong and should be:
```suggestion
		if (e.getKeyCode() == KeyEvent.VK_CONTROL && config.disableOnCtrl() && !config.onShiftOnly())
```
Since ctrl to disable only makes sense when shift-only is off, otherwise you would have to hit both shift + ctrl at once to enable and then disable which doesn't make sense.

Reply:
Now it only works in pvp when you enable onShiftOnly